starter-code/src/App.css

```diff
@@ -0,0 +1,76 @@
+.App{
+
+}
+/* body{
+    margin: 0%;
+    padding: 0%;
+} */
+
+
+.firstpart{
+    background-color: #1e2536;
+    height: 800px;
+    margin-left: 15%;
```

Comment:
you don't need to add the margin 15%. In the example, the first part takes up the full width of the screen. It's not super clear on the exercise, but if you click the image from the lab you can see that it's 100% of the screen  with